index.bs

```diff
@@ -608,77 +633,129 @@ interface ScreenDetailed : Screen {
--->
+This specification defines a [=/default powerful feature=] that is identified by the [=powerful feature/name=] "<dfn for="PermissionName" export enum-value>window-placement</dfn>".
+
+The [[permissions]] API provides a uniform way for websites to query the state of their permissions.
+
+Issue: Add {{PermissionName/window-placement}} to {{PermissionName}} in [[permissions]].
+
+Issue: Define behavior of cached objects and method steps when the permission is revoked. (See [#80](https://github.com/webscreens/window-placement/issues/80))
+
+<!-- ====================================================================== -->
+## Permission Policy integration ## {#api-permission-policy-integration}
+<!-- ====================================================================== -->
+
+This specification defines a [=/policy-controlled feature=] identified by the string "<dfn for="permission-policy">window-placement</dfn>". Its [=/default allowlist=] is `"self"`. See [[permissions-policy]] and its list of [Experimental Features](https://github.com/w3c/webappsec-permissions-policy/blob/main/features.md#experimental-features).
+
+Note: A [=/document=]’s permissions policy determines whether any content in that document is allowed to obtain {{ScreenDetails}} or place content on specific screens. If disabled, promises returned by {{Window/getScreenDetails}} will be rejected, and requests to place content on specific screens will be clamped to the [=/current screen=].
+
+Issue: Move [=permission-policy/window-placement=] to [Proposed](https://github.com/w3c/webappsec-permissions-policy/blob/main/features.md#proposed-features) or [Standardized](https://github.com/w3c/webappsec-permissions-policy/blob/main/features.md#proposed-features) feature lists when appropriate.
 
 <!-- ====================================================================== -->
 # Security Considerations # {#security}
 <!-- ====================================================================== -->
 
-Issue: Write this section.
+This specification enables sites to place content on specific screens, which may pose limited new security risks. Sites may attempt to prominently display sensitive content on unexpected screens, surreptitiously display undesirable content on less conspicuous screens, or otherwise place content on specific screens to act in deceptive, abusive, or annoying manners.
+
+To help mitigate such risks, cross-screen placement capabilities are restricted to secure contexts, require explicit user permission (where prompting is only possible with transient user activation), and are subject to permission policy. If any of these requirements are not met, placement requests may be denied or clamped to the [=/current screen=], matching pre-existing behavior of some user agents. User agents can generally measure and otherwise intervene when sites request any new capabilities.
```

Comment:
```
[…] matching pre-existing behavior of some user agents.
```

Curious about the word "some" here. I thought this was universal behavior?!

Reply:
Ah, I see. It was just me being curious, nothing for the spec. Thanks for the additional details.